srml/contracts/src/exec.rs

```diff
@@ -154,6 +154,9 @@ pub trait Ext {
 	/// Returns a reference to the timestamp of the current block
 	fn now(&self) -> &MomentOf<Self::T>;
 
+	/// Returns the minimum balance that is required for creating an account.
+	fn minimum_balance(&self) -> BalanceOf<Self::T>;
```

Comment:
Why not call it `existential_deposit` as it is called everywhere else in Substrate?

Reply:
Tbh I think `minimum_balance` is the better name here. Name is explained above in the PR comment.